psh/echo/echo.c

```diff
@@ -0,0 +1,97 @@
+/*
+ * Phoenix-RTOS
+ *
+ * echo - display a line of text
+ *
+ * Copyright 2022 Phoenix Systems
+ * Author: Aleksander Kaminski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "../psh.h"
+
+
+static void psh_echoinfo(void)
+{
+	printf("display a line of text");
+}
+
+
+static void psh_echo_help(const char *prog)
+{
+	printf("Usage: %s [options] [string]\n", prog);
+	printf("  -h:  shows this help message\n");
+}
```

Comment:
Just an idea: maybe extend help about `$?` feature? As we currently have only one magic echo usage it may be handy

Reply:
Ok, I'm not sure about it's format, but I added it